examples/vae/utils.py

```diff
@@ -0,0 +1,90 @@
+"""
+    This code is created with reference to torchvision/utils.py.
+
+    Modify: torch.tensor -> jax.numpy.DeviceArray
+
+    If you want to know about this file in detail, please visit the original code:
+        https://github.com/pytorch/vision/blob/master/torchvision/utils.py
+"""
+import math
+import jax
+import jax.numpy as np
+
+irange = range
+
+
+def make_grid(ndarray, nrow=8, padding=2,
+              normalize=False, range=None, scale_each=False, pad_value=0.0):
+    """Make a grid of images.
+    Args:
+        ndarray (array_like or list): 4D mini-batch ndarray of shape (B x H x W x C)
+            or a list of images all of the same size.
+        nrow (int, optional): Number of images displayed in each row of the grid.
+            The final grid size is ``(B / nrow, nrow)``. Default: ``8``.
+        padding (int, optional): amount of padding. Default: ``2``.
+        normalize (bool, optional): If True, shift the image to the range (0, 1),
+            by the min and max values specified by :attr:`range`. Default: ``False``.
+        range (tuple, optional): tuple (min, max) where min and max are numbers,
+            then these numbers are used to normalize the image. By default, min and max
+            are computed from the tensor.
+        scale_each (bool, optional): If ``True``, scale each image in the batch of
+            images separately rather than the (min, max) over all images. Default: ``False``.
+        pad_value (float, optional): Value for the padded pixels. Default: ``0``.
+    """
+    if not (isinstance(ndarray, np.DeviceArray) or
```

Comment:
I think it is better to check `isinstance(ndarray, jnp.ndarray)`

Reply:
Fix it